azurerm/internal/services/cosmos/cosmosdb_gremlin_graph_resource.go

```diff
@@ -102,14 +103,15 @@ func resourceCosmosDbGremlinGraph() *schema.Resource {
 							Default:  true,
 						},
 
+						// case change in 2021-01-15, issue https://github.com/Azure/azure-rest-api-specs/issues/14051
 						"indexing_mode": {
```

Comment:
could we add a comment 
```suggestion
						// todo: change to SDK constants and remove translation code in 3.0
						"indexing_mode": {
```

for the constants we've done this with?